src/message.ts

```diff
@@ -613,6 +625,40 @@ export class Message implements Sayable {
   // }
 
   /**
+   * @param sendTo UserId/RoomId/Room/Contact
+   */
+  public forward(room: Room): Promise<any>
+  public forward(contact: Contact): Promise<any>
+  public forward(id: string): Promise<any>
+  public forward(sendTo: string|Room|Contact): Promise<any> {
+    const m = <MsgRawObj>this.rawObj
+    const newMsg = <MsgRawObj>{}
+    let id = ''
+    if (sendTo instanceof Room || sendTo instanceof Contact) {
+      id = sendTo.id
+    } else {
+      id = sendTo
+    }
+    newMsg.ToUserName = id
+    newMsg.FromUserName = config.puppetInstance().userId || ''
+    newMsg.isTranspond = true
+    newMsg.MsgIdBeforeTranspond = m.MsgIdBeforeTranspond || m.MsgId
+    newMsg.MMSourceMsgId = m.MsgId
+    newMsg.sendByLocal = false
+    newMsg.Content = UtilLib.unescapeHtml(m.Content.replace(/^@\w+:<br\/>/, ''))
+    newMsg.MMActualSender = config.puppetInstance().userId || ''
+    if (m.MMSendContent)
+      newMsg.MMSendContent = m.MMSendContent.replace(/^@\w+:\s/, '')
+    if (m.MMDigest)
+      newMsg.MMDigest = m.MMDigest.replace(/^@\w+:/, '')
+    if (m.MMActualContent)
+      newMsg.MMActualContent = UtilLib.stripHtml(m.MMActualContent.replace(/^@\w+:<br\/>/, ''))
+
+    return config.puppetInstance()
+      .forward(m, newMsg)
```

Comment:
Could we merge `newMsg` to `m` and just use `.forward(m)` at here?

Because I noticed that the `newMsg` is used as the `patchData` in `wechaty-bro`, and just do the `m = Object.assign(m, patchData)` later.